skrf/media/rectangularWaveguide.py

```diff
@@ -121,28 +121,27 @@ def __repr__(self):
 
     
     @classmethod
-    def from_Z0(cls,frequency, Z0,f, m=1,n=0,ep_r=1, mu_r=1, **kw):
```

Comment:
Well, `m=1, n=0` is hard-coded on line 144. I think we discussed it a little bit, and it mainly makes sense for the fundamental mode. Otherwise you would have to deduce two parameters from one given parameter which leaves some ambiguity. So in my opinion having it as a parameter in the function is not so useful. That's also the reason why I updated the docstring documenting the function. But I'm still open for discussion^^